src/app/shared/components/disk-offering/disk-offering-selector/disk-offering-selector.component.ts

```diff
@@ -72,4 +99,14 @@ export class DiskOfferingSelectorComponent implements ControlValueAccessor {
         }
       });
   }
+
+  private setMaxSizeValue() {
+    const customDiskOfferingMaxSize = this.authService.getCustomDiskOfferingMaxSize();
+    this.min = this.min ? this.min : this.authService.getCustomDiskOfferingMinSize();
```

Comment:
Why don't you use `customDiskOfferingMaxSize`?
Is this valid if `this.min` = 0? If so, you can replace this with `this.min || customDiskOfferingMaxSize`

Reply:
the reason is:
 @tamazlykar doesn't like construction like this.  He thinks that ternary operator more readable.
But for me, it doesn't matter and because of this,  I use ternary operator :)